distrho/src/DistrhoPluginAU.cpp

```diff
@@ -221,6 +221,8 @@ class PluginAU : public AUEffectBase
             destBuffer[i] = (float *)outBuffer.mBuffers[i].mData;
         }
 
+        updateSampleRate();
```

Comment:
this is not something that should be done in every process, but I'll handle this case later on